### PR TITLE
[IMP] mail: introduce NotificationRequestView/headerText

### PR DESCRIPTION
**Pull request by alexkuhn**
Task-2831082

### CODE REVIEW

File: addons/mail/static/src/models/notification_request_view.js [hunk 1]
```diff
@@ -1,12 +1,29 @@
 /** @odoo-module **/
 
 import { registerModel } from '@mail/model/model_core';
-import { one } from '@mail/model/model_field';
+import { attr, one } from '@mail/model/model_field';
+
+import { sprintf } from '@web/core/utils/strings';
```
Comment (oomsveta) [inline addons/mail/static/src/models/notification_request_view.js:6]: Is it usual to add a new line to separate the imports of different modules? I don't remember.

File: addons/mail/static/src/models/notification_request_view.js [hunk 2]
```diff
@@ -1,12 +1,29 @@
...
     name: 'NotificationRequestView',
     identifyingFields: ['notificationListViewOwner'],
+    recordMethods: {
+        /**
+         * @private
+         * @returns {string}
+         */
+        _computeHeaderText() {
+            return sprintf(
+                this.env._t("%s has a request"),
```
Comment (oomsveta) [inline addons/mail/static/src/models/notification_request_view.js:18]: You may want to name the placeholder to provide some more context to the translators. I can think of some languages that have different rules depending on whether the subject is a human, an animal, an inanimate object… and for which this kind of information matters.

**Review by oomsveta — CHANGES_REQUESTED** [review]

File: addons/mail/static/src/models/notification_request_view.js (hunk 1 above)
Comment (alexkuhn) [reply addons/mail/static/src/models/notification_request_view.js:6]: Yes

File: addons/mail/static/src/models/notification_request_view.js (hunk 2 above)
Comment (alexkuhn) [reply addons/mail/static/src/models/notification_request_view.js:18]: Change done

**Review by oomsveta — APPROVED** [review]

**Comment by alexkuhn** [comment]
Merged in grouped PR: https://github.com/odoo/odoo/pull/91000

**Pull request closed without merging**